runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -3550,9 +3555,6 @@ static bool updateCHTable(J9VMThread * vmThread, J9Class  * cl)
    TR_PersistentCHTable * table = 0;
    if (TR::Options::getCmdLineOptions()->allowRecompilation()
       && !TR::Options::getCmdLineOptions()->getOption(TR_DisableCHOpts)
-#if defined(J9VM_OPT_JITSERVER)
-      && compInfo->getPersistentInfo()->getRemoteCompilationMode() != JITServer::SERVER
```

Comment:
Why is this code removed. The idea is that, we don't want to update the CHTable at the server because it is not used for compilation anyway.

Reply:
I think `persistentMemory->getPersistentInfo()->getRuntimeAssumptionTable()->init()` is the only place where we call `updateCHTable()` at the server. Since that's disabled, `updateCHTable()` is no longer called at the server.

Reply:
To be sure let's add an ASSERT_FATAL